docs/src/content/pages/conditional-field.mdoc

```diff
@@ -1,6 +1,92 @@
 ---
 title: Conditional field
 ---
+The `conditional` field lets you setup a&nbsp;`discriminant`&nbsp;for a condition that can be either a&nbsp;`checkbox`&nbsp;or a&nbsp;`select`&nbsp;field with a few options.
+
+You can then display an entirely different set of fields based on the condition, by providing a key for each condition:
+
+- `true`/`false`&nbsp;if you used a&nbsp;`checkbox`
+- each possible value if you used a&nbsp;`select`
+  - nested
```

Comment:
Is this nested item meant to be there?

Reply:
HAH!

This was me testing @lukebennett88's bug report where nested bullet points disappear on save:

https://thinkmill.slack.com/archives/C04CLQBUF29/p1685257089354049

I just forget the fact that they disappear from the Admin UI, but **are** actually persisted in the generated file 🤣

Reply:
I just did a commit removing it 👍